stories/cosmoz-autocomplete.stories.js

```diff
@@ -1,122 +1,192 @@
 import { html } from 'lit-html';
-import '../src/autocomplete/';
+import '../dist/autocomplete/';
```

Comment:
You can revert this to `../src/autocomplete` for now. The build will replace `src/` with `dist/`.